src/components/KYCWall/BaseKYCWall.js

```diff
@@ -123,9 +124,9 @@ class KYCWall extends React.Component {
         }
         if (!isExpenseReport) {
             // Ask the user to upgrade to a gold wallet as this means they have not yet gone through our Know Your Customer (KYC) checks
-            const hasGoldWallet = this.props.userWallet.tierName && this.props.userWallet.tierName === CONST.WALLET.TIER_NAME.GOLD;
-            if (!hasGoldWallet) {
-                Log.info('[KYC Wallet] User does not have gold wallet');
+            const hasActivatedWallet = this.props.userWallet.tierName && _.contains([CONST.WALLET.TIER_NAME.GOLD, CONST.WALLET.TIER_NAME.PLATINUM], this.props.userWallet.tierName);
```

Comment:
NAB, we can have a separate method to check if wallet is one of those specific tiers

Reply:
I had the same thought but I don't believe we have a model (or some sort of lib) class for Wallets?